### ISSUE TITLE
Does not support localised image resources

### ISSUE THREAD

**Issue by stevenoglesby** [nicklockwood/StandardPaths#14]
Hi Nick,

I think this is an issue, but it is the first time I have had to localise image resources so I might just be missing something.

If an image is localised then it is stored in Base.lproj, fr.lproj etc folders inside the MainBundle of the app.
All the lookup in normalizedPathForFile:ofType seems to be based on the resourcePath which is just got from the MainBundle. So it does not find then find any of the images when it gets with fileExistsAtPath:.

So it then eventually falls back to just the original filename for a normal image lookup.

I am happy to have a bit more of a poke around if you think this is a real issue.

Thanks,
Steven

**Comment by nicklockwood** [comment]
Yes, it's likely this is broken. I had to switch to using resourcePath to fix an issue back in iOS7, but this probably broke localised resource files.

**Comment by stevenoglesby** [comment]
Hi,

OK. I am going to put something together using [NSBundle preferredLocalizations] to try to check the paths for the localisations after having checked the resourcePath first (as guess localised images are reasonably rare case).
It might not be optimal but hopefully I can get something working.

**Comment by stevenoglesby** [comment]
Have created pull request for you to take a look at.

https://github.com/nicklockwood/StandardPaths/pull/15
